libraries/proguard-rx-java.pro

```diff
@@ -1,3 +1,8 @@
+# RxJava 1.0.14
+-keep class rx.internal.util.unsafe.** {
```

Comment:
The issue can be also be fixed with this:
```
-keepclassmembers class rx.internal.util.unsafe.*ArrayQueue*Field* {
    long producerIndex;
    long consumerIndex;
}
-keepclassmembers class rx.internal.util.unsafe.BaseLinkedQueueProducerNodeRef {
    long producerNode;
    long consumerNode;
}
```
found here: https://github.com/ReactiveX/RxJava/issues/3097#issuecomment-135125272. This will also allow ProGuard to be run on more classes.